Scripting/UTM.sdef

```diff
@@ -92,6 +92,17 @@
             </parameter>
         </command>
         
+        <command name="import" code="coreimpo" description="Import a new virtual machine from a file.">
+            <cocoa class="UTMScriptingImportCommand"/>
+            <parameter name="new" code="imcl" type="type" description="Specify 'virtual machine' here.">
```

Comment:
Feels weird to have a parameter that doesn't do anything and also fails if you pass in another type. Is is possible to rename the command to something like "import virtual machine" and have only one parameter?

Reply:
This is similar to 
`make new virtual machine with properties ...`

Since both import and make creates new object (NSobject), I have used same syntax and code. 

The new keyword is a convention in Apple script to indicate creation of new object and clearly implies you are creating and importing a new virtual machine in UTM. 

Also, this command will always create new VM, rather than VM short cut.